api/proto/config/devmodel.proto

```diff
@@ -4,6 +4,7 @@
 syntax = "proto3";
 
 import "evecommon/devmodelcommon.proto";
+import "config/devcommon.proto";
```

Comment:
This line should be in the first commit and not in the second one with the generated and vendored files.